scripts/docker

```diff
@@ -356,6 +356,7 @@ else
             ${VOLUME_PREFIX}couchdb \
             ${VOLUME_PREFIX}elasticsearch2 \
```

Comment:
Wonder if this line can be removed? ES2 should be long gone. @AmitPhulera?

Reply:
Agreed. I think both elasticsearch2 and elasticsearch5 should be removed.

Reply:
Addressing in https://github.com/dimagi/commcare-hq/pull/36408. We still have some code that is using es5, so this change makes me slightly more nervous than this current PR, hence forking it off